src/Transports.AspNetCore/WebSockets/GraphQLWs/SubscriptionServer.cs

```diff
@@ -0,0 +1,254 @@
+using GraphQL.Transport;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets.GraphQLWs;
+
+/// <inheritdoc/>
+public class SubscriptionServer : BaseSubscriptionServer
+{
+    private readonly IWebSocketAuthenticationService? _authenticationService;
+
+    /// <summary>
+    /// The WebSocket sub-protocol used for this protocol.
+    /// </summary>
+    public const string SubProtocol = "graphql-transport-ws";
```

Comment:
It looks like you flipped protocols.

Reply:
It's stupid, but correct.  The repo at https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md (with npm package named `graphql-ws`) uses the sub-protocol `graphql-transport-ws`.  The repo at https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md (with npm package named `subscriptions-transport-ws`) uses the sub-protocol `graphql-ws`.  So here I've named the folders and namespaces according to the npm package name, which seems to be the most common naming (as they are both just called "GraphQL over WebSockets") -- and then have a const for the correct sub-protocol to use.

Reply:
WOW! We definitely should leave a comment about this so that the reader does not have an impression of an error (like mine).

Reply:
Agreed.

Reply:
```suggestion
    /// <remarks>
    /// Please note that the correct sub-protocol for the
    /// <see href="https://github.com/enisdenjo/graphql-ws">graphql-ws</see>
    /// protocol is <c>graphql-transport-ws</c> as is defined here.
    /// </remarks>
    public const string SubProtocol = "graphql-transport-ws";
```

Reply:
👍